test/schemas/v0/tests.schema.json

```diff
@@ -118,6 +118,9 @@
         "src": {
           "$ref": "#/$defs/src"
         },
+        "bidiIsolation": {
```

Comment:
https://github.com/unicode-org/conformance/blob/main/schema/message_fmt2/testgen_schema.json will need to be updated to reflect this change.